torch/utils/tensorboard/writer.py

```diff
@@ -293,7 +293,7 @@ def get_logdir(self):
         return self.log_dir
 
     def add_hparams(
-        self, hparam_dict, metric_dict, hparam_domain_discrete=None, run_name=None
+        self, hparam_dict, metric_dict, global_step=None, hparam_domain_discrete=None, run_name=None
```

Comment:
can you move this to the end of the arg list (and maybe make it kwarg only)? As written this is BC breaking because someone could have passed these args in positionally.

Reply:
I dropped the commit that made that change